src/ResponsiveImages/ResponsiveImageGenerator.php

```diff
@@ -64,18 +64,30 @@ public function generateResponsiveImagesForConversion(Media $media, Conversion $
         $media = $this->cleanResponsiveImages($media, $conversion->getName());
 
         foreach ($this->widthCalculator->calculateWidthsFromFile($baseImage) as $width) {
-            $this->generateResponsiveImage($media, $baseImage, $conversion->getName(), $width, $temporaryDirectory);
+            $this->generateResponsiveImage($media, $baseImage, $conversion->getName(), $this->getConversionQuality($conversion), $width, $temporaryDirectory);
         }
 
         $this->generateTinyJpg($media, $baseImage, $conversion->getName(), $temporaryDirectory);
 
         $temporaryDirectory->delete();
     }
 
+    private function getConversionQuality(Conversion $conversion = null): int
+    {
+        $defaultConversionQuality = 90;
+
+        if (! $conversion) {
+            return $defaultConversionQuality;
+        }
+
+        return $conversion->getManipulations()->getManipulationArgument('quality') ?: $defaultConversionQuality;
+    }
+
     public function generateResponsiveImage(
         Media $media,
         string $baseImage,
         string $conversionName,
+        int $conversionQuality,
```

Comment:
Addin a parameter here is a breaking change.

Reply:
Sure thing, I'll fix that shortly, likely just make it an optional parameter.

Reply:
See the [newest commit](https://github.com/spatie/laravel-medialibrary/pull/1881/commits/bd3b36a27380628d38ed21baa4e1f37ddb41ba35) that (hopefully) achieved the changes you requested.